features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos1
+    Then the image is listed as active
+    And this image has non-empty ID
+    And its ID can be used to show the image info
+    When I delete the image
+    Then it is no longer listed
+
+  @create_xen_hvm_image
+  Scenario: Creation of XEN HVM image
+    Given XEN HVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos1
+    Then the image is listed as active
+    And this image has non-empty ID
+    And its ID can be used to show the image info
+    When I delete the image
```

Comment:
Do we need to test deletion for all the different image types, or maybe is one enough?